app/views/records/edit_fields/_creator.html.erb

```diff
@@ -1,5 +1,6 @@
 <% template = f.object.model.class.to_s.underscore %>
 <% types = HykuAddons::NameTypeService.new.select_active_options.flatten.uniq! %>
+<% required = f.object.class != Hyrax::Forms::CollectionForm %>
```

Comment:
I know I suggested the check against the class before but I think it would be better to ask the form:
```suggestion
<% required = f.required?(:creator) %>
```

Reply:
The change is made. I opted to use the `f.required?(:creator)` inline to make it a little more self explaining.